bpf/Makefile

```diff
@@ -96,7 +96,7 @@ MAX_BASE_OPTIONS += -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 \
 	-DENABLE_HOSTPORT=1 -DENABLE_LOADBALANCER -DENABLE_DSR_HYBRID=1 \
 	-DENABLE_DSR_ICMP_ERRORS=1 -DENABLE_IPV4_FRAGMENTS=1
 ifeq ("$(KERNEL)","54")
-MAX_BASE_OPTIONS += -DENABLE_BANDWIDTH_MANAGER=1
+MAX_BASE_OPTIONS += -DENABLE_BANDWIDTH_MANAGER=1 -DNETFILTER_COMPAT_MODE=1
```

Comment:
That should be added to MAX_HOST_OPTIONS instead as commented in https://github.com/cilium/cilium/pull/15354#discussion_r647434520.

Reply:
It looks like it should also be defined for `KERNEL=netnext`.

Reply:
Sure will revert only to `MAX_HOST_OPTIONS`. I thought may be good idea to add in `MAX_BASE_OPTIONS` to test for bpf files also especially [bpf_lxc] .